### PR TITLE
Add Drop-down Widget

### PR DESCRIPTION
**Pull request by Antol**
## Description
This pull request adds a new Drop-down Widget to the Grist custom widget collection. The widget provides a simple and compact drop-down menu that can be populated with options from a selected column in a Grist document.

### Features
- Displays options from a selected column in a drop-down menu
- Acts as a source for selection in other Grist widgets
- Compact design for efficient use of space
- Handles error states and empty option sets
- Syncs with Grist cursor position and theme

### Files Added
- `drop-down/index.html`: Main HTML structure
- `drop-down/script.js`: Widget functionality
- `drop-down/package.json`: Widget metadata
- `drop-down/README.md`: Usage instructions and widget description

### How to Use
1. Add the widget to a Grist document
2. Select the column to use for options in the widget configuration
3. The drop-down will populate with values from the selected column
4. Selecting an option will update the cursor position in linked widgets

This widget aims to provide a straightforward and useful tool for Grist users who need a compact way to navigate through options in their documents.

### CODE REVIEW

**Comment by berhalak** [comment]
Hi @Antol, thanks for contributing.

Can you describe the purpose of this widget, what kind of problems it solves? I also noticed that the dropdown has 100% height and width, which looks surprising, was this your intention?

**Comment by Antol** [comment]
@berhalak I will quote the August newsletter "Antol Peshkov has created a simple custom dropdown widget that does exactly what it sounds like." 
Why is 100% height and width surprising? It would take only a small piece of a page to work as a selector.
600x486

File: drop-down/package.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+  "name": "@antol/widget-drop-down",
+  "description": "Simple drop-down widget for Grist.",
+  "homePage": "https://github.com/Antol/grist-widget",
+  "version": "0.0.1",
+  "grist": {
+    "name": "Drop-down Widget",
+    "url": "https://antol.github.io/grist-widget/drop-down/index.html",
```
Comment (berhalak) [inline drop-down/package.json:8]: Can you change the URL to point to this site?

Comment (Antol) [reply drop-down/package.json:8]: Yes, of course

**Pull request merged**